frontend/src/component/project/Project/ProjectForm/ProjectForm.tsx

```diff
@@ -113,6 +120,30 @@ const ProjectForm: React.FC<IProjectForm> = ({
                         </>
                     }
                 />
+                <ConditionallyRender
+                    condition={Boolean(projectModeFlag)}
```

Comment:
this flag should be enough w/o additional enterprise check since the parent component hides create project screen